code/game/objects/items/weapons/storage/uplink_kits.dm

```diff
@@ -336,3 +336,17 @@ obj/item/weapon/storage/box/syndie_kit/cheaptide
 	new /obj/item/weapon/card/emag(src)
 	new /obj/item/weapon/glue(src)
 	return
+
+
+//Syndicate Experimental Gear
+//Contains unique gear not found anywhere else
+/obj/item/weapon/storage/box/syndicate_experimental/New()
+	..()
+	var/selection = pick("damocles", "bomber vest", "bike horn")
+	switch(selection)
+		if("damocles")
+			new /obj/item/weapon/damocles
```

Comment:
Did you test this ? This would spawn the thing in nullspace and it would be unreachable.
If you want it in the box, use `new /obj/thing(src)`.